openedx/features/edly/templates/edly/edx_ace/commentvotenotification/email/body.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <p>
+                {% blocktrans trimmed %}
+                    {{ voter_name }} gives you thumbs up on your following comment of post <b>{{ thread_title }}</b>:
```

Comment:
`{{ voter_name }} gives you thumbs up on your following comment of post`
-> 
`{{ voter_name }} likes your following comment on the post with title "<b>{{thread_title }}</b>:`

Reply:
done